# Solución de examen 1

## Situación 3

Antonio, un profesor de matemáticas participará en el Tour de Francia (competencia de ciclismo) donde usará su bicicleta de ruedas cuadradas. La longitud de uno de los recorridos es de $192\ \rm{km}$.

**9. Si las ruedas cuadradas tienen $50\ \rm{cm}$ de lado, ¿cuántas vueltas tendrá que dar una de las ruedas para llegar a la meta?**

El perímetro de un cuadrado
$$P=4L=4(50\ \rm{cm})=200\ \rm{cm}$$

Usando regla de tres simple varias veces:

$$192\ \rm{km}=192000\ \rm{m}=19200000\ \rm{cm}$$

$\text{Número de vueltas}=\dfrac{\text{Longitud de la pista}}{\text{Perímetro de la rueda cuadrada}} =96000$

In [1]:
19200000/200

96000.0

a) 7680

b) 76800

c) 96000

d) 192000

**Solución:** c)

**10. Marie, una ciclista francesa, también se encuentra en la competencia. Su bicicleta tiene ruedas circulares de $50\ \rm{cm}$ de diámetro. Si Marie pedalea hasta lograr que su rueda gire a $45^\circ$ cada centésima de segundo, ¿cuánto tiempo tardará en llegar a la meta?**

Primero convertimos el ángulo a radianes para saber la longitud de arco.

$$ 45^\circ=\dfrac{45}{180}\pi\ \rm{rad}=\dfrac{\pi}{4}\ \rm{rad}$$

In [2]:
45*pi/180

0.7853981633974483

$$ \text{longitud de arco}=\theta\cdot R $$

$$ =\left(\dfrac{\pi}{4}\right)\cdot(25\ \rm{cm}) $$

$$ =19.63475cm $$

In [3]:
0.78539*25

19.63475

Toda la pista mide $192\ \rm{km}=19200000\ \rm{cm}$

$$\text{Número de veces que la rueda giró }45^\circ=\dfrac{\text{Longitud de la pista}}{\text{Longitud de arco de la rueda}}=977858.13 $$

In [4]:
19200000/19.63475

977858.1341753778

$$\text{Tiempo}=(\text{Número de veces que la rueda giró }45^\circ)\times(\text{Tiempo que tarda la rueda en girar }45^\circ)=9778.58 $$

In [5]:
977858.1341753778*1/100

9778.581341753777

Para llegar a la meta Marie se tarda $9778.58\ \rm{s}$

Usamos regla de tres para pasar los segundos a horas:

$$ 9778.58\ \rm{s}=9778.58\ \rm{s}\left(\dfrac{1\ \rm{min}}{60\ \rm{s}}\right)\left(\dfrac{1\ \rm{hora}}{60\ \rm{min}}\right)=2.71\ \rm{horas} $$

In [6]:
9778.58/60

162.97633333333334

In [7]:
162.97/60

2.7161666666666666

Para llegar a la meta Marie se tarda $2.71\ \rm{horas}$

a) Menos de 2 horas

b) Más de 2 horas pero menos 4 horas

c) Más de 4 horas pero menos 6 horas

d) Ninguna de las anteriores

**Solución:** b)

**11. Marie cambia las ruedas por otras que tienen un rin de $55\ \rm{cm}$ de diámetro con una llanta de $3\ \rm{cm}$ de ancho. ¿Cuánta distancia recorrerá la nueva rueda al dar una vuelta?**

$$ P=C=2\pi\cdot R=\pi\cdot D $$

El nuevo radio sería $30.5\ \rm{cm}$

El  nuevo diámetro sería $61\ \rm{cm}$

$$ P=C=61\pi\ cm $$

a) $122\pi\ \rm{cm}$

b) $116\pi\ \rm{cm}$

c) $61\pi\ \rm{cm}$

d) $58\pi\ \rm{cm}$

**Solución:** c)

**12. Marie quiere pintar los rines de sus nuevas ruedas de color naranja. Cada rayo (sector circular) tiene un ángulo de $30^\circ$ desde el centro de la rueda. Si la pintura cuesta $0.5$ euros el mililitro y un mililitro de pintura rinde para $130 \rm{cm}^2$, ¿cuánto gastará para pintar ambas cara de sus dos rines?**

<img src="rines.png" align="middle" width="400">

El área de un sector circular es

$$ A=\dfrac{\theta R^2}{2} $$

Convertimos el ángulo a radianes.

$$ 180^\circ=\dfrac{\pi}{6}\ \rm{rad} $$

In [9]:
π/6

0.5235987755982988

$$ A=\dfrac{\dfrac{\pi}{6} (27.5\ \rm{cm})^2}{2} $$

In [10]:
0.5235987755982988*(27.5)^2/2

197.98578702310675

El área del sector circular es $197.98\ \rm{cm}^2$

Como hay $4$ rayos, $2$ caras de cada rin, y $2$ rines, tenemos que multiplicar el área del sector circular por $16$.

In [11]:
16*197.98578702310675

3167.772592369708

El área total a pintar es $3167.77\ \rm{cm}^2$

In [12]:
3167.772592369708/130

24.367481479766983

De aquí hay que usar pura regla de tres:

La pintura necesaria es $24.36\ \rm{mL}$

In [13]:
24.367481479766983*0.5

12.183740739883492

La chamba salió en $12.18$ euros.

a) Menos de $5$ euros

b) Más de $5$ euros pero menor de $10$ euros

c) Más de $10$ euros pero menor de $15$ euros

d) Más de $15$ euros

**Solución:** c)

## Situación 1

**1. La diagonal de un cuadrado mide $x$ unidades. Su área, en unidades cuadradas, es**

<img src="fractal1.png" align="middle" width="160">

$$ L^2+L^2=x^2 $$

$$ 2L^2=x^2 $$

$$ L^2=\dfrac{x^2}{2} $$

$$ L=\sqrt{\dfrac{x^2}{2}} $$

El área de un cuadrado
$$A=L\cdot L=L^2=\dfrac{x^2}{2}$$

a) $\dfrac{x}{\sqrt{2}}$

b) $\sqrt{2}x^2$

c) $\dfrac{x^2}{2}$

d) $\dfrac{\sqrt{3}}{4}x$

**Solución:** c)

**2. Para la figura que se muestra, la proporción del área del triángulo con respeto al área del cuadrado es**

<img src="fractal2.png" align="middle" width="175">

a) $\dfrac{1}{4}$

b) $\dfrac{1}{8}$

c) $\dfrac{1}{9}$

d) $\dfrac{1}{16}$

**Solución:** 

**3. El área sombreada de la siguiente figura, en unidades cuadradas, es**

<img src="fractal3.png" align="middle" width="300">

a) $2x^2+\dfrac{1}{4}$

b) $\dfrac{x^2}{2}+\dfrac{1}{8}$

c) $\dfrac{7}{16}x^2$

d) $\dfrac{3}{4}x^2$

**Solución:** 

**4. El área sombreada de la siguiente figura, en unidades cuadradas, es**

<img src="fractal4.png" align="middle" width="300">

a) $\dfrac{3}{2}x^2$

b) $\dfrac{5}{4}x^2$

c) $\dfrac{7}{8}x^2$

d) $\dfrac{3}{4}x^2$

**Solución:** 

# Álgebra para analizar objetos geométricos

# Clase 1

### Ejes coordenados

Eje $x$ (eje de las abscisas)

Eje $y$ (eje de las ordenadas)

**¿Cómo nos movemos en el plano cartesiano?**

Para empezar a movernos en el plano cartesiano tenemos que partir del origen $O(0,0)$

Si las coordenadas son $(x,y)$, la $x$ nos indica cuántas unidades movernos sobre el eje de las $x$ (si es positivo nos movemos hacia la derecha; si es negativo, hacia la izquierda).

La segunda coordenada (la ordenada $y$) nos indica cuántas unidades movernos sobre el eje de las $y$ (si es positivo nos movemos hacia arriba; si es negativo, hacia abajo).

$P_1(1,3)$

$P_2(-2,2)$

$P_3(-4,-3)$

$P_4(2,-4)$

Cuadrantes:
* Primer cuadrante $(+,+)$
* Segundo cuadrante $(-,+)$
* Tercer cuadrante $(-,-)$
* Cuarto cuadrante $(+,-)$


## Distancia

Si tenemos dos puntos $P(x_1,y_1)$ y $Q(x_2,y_2)$, la longitud del segmento entre estos puntos es

$$ \overline{PQ}=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

* ***Ejemplo:*** Determine la distancia entre los puntos $A(-2,1)$, y $B(1,5)$

$$ \overline{AB}=\sqrt{[(-2)-(1)]^2+[(1)-(5)]^2} $$

$$ =\sqrt{[-3]^2+[-4]^2} $$

$$ =\sqrt{9+16} $$

$$ =\sqrt{25} $$

$$ =5 $$

Si tenemos dos puntos $P(x_1,y_1)$ y $Q(x_2,y_2)$, la longitud del segmento entre estos puntos es

$$ \overline{PQ}=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

* ***Ejemplo:*** Determine el punto equidistante entre los puntos $A(-2,3)$, $B(4,1)$, y $C(1,7)$

El punto del problema es encontrar El cuarto que será equidistante a los otros tres puntos es $P(x,y)$.

¿Cuál es la distancia entre $A$ y $P$?

$$\overline{AP}=\sqrt{[(-2)-x]^2+[(3)-y]^2}$$

$$\overline{AP}=\sqrt{(-2-x)^2+(3-y)^2}$$

¿Cuál es la distancia entre $B$ y $P$?

$$\overline{BP}=\sqrt{[(4)-x]^2+[(1)-y]^2}$$

$$\overline{BP}=\sqrt{(4-x)^2+(1-y)^2}$$

¿Cuál es la distancia entre $C$ y $P$?

$$\overline{CP}=\sqrt{[(1)-x]^2+[(7)-y]^2}$$

$$\overline{CP}=\sqrt{(1-x)^2+(7-y)^2}$$

**1ra ecuación**

$$ \overline{AP}=\overline{BP} $$

$$ \sqrt{(-2-x)^2+(3-y)^2}=\sqrt{(4-x)^2+(1-y)^2} $$

Elevamos toda la ecuación al cuadrado para eliminar las raíces de cada lado:

$$ (-2-x)^2+(3-y)^2=(4-x)^2+(1-y)^2 $$

Elevamos cada binomio al cuadrado:

$$ [(-2)^2+2(-2)(-x)+(-x)^2]+[(3)^2+2(3)(-y)+(-y)^2]=[(4)^2+2(4)(-x)+(-x)^2]+[(1)^2+2(1)(-y)+(-y)^2] $$

$$ [4+4x+x^2]+[9-6y+y^2]=[16-8x+x^2]+[1-2y+y^2] $$

$$ 4x+x^2-6y+y^2+8x-x^2+2y-y^2=16+1-4-9 $$

$$ 4x-6y+8x+2y=16+1-4-9 $$

$$ 12x-4y=5 $$

**2da ecuación**

$$ \overline{BP}=\overline{CP} $$

$$ ? $$

**Tarea:** Terminen el ejercicio y encuentren que el punto es $P(3/2,7/2)$.

* ***Ejemplo:*** Determina si el triángulo formado por los puntos $A(0,0)$, $B(6,5)$, y $C(1,6)$ es escaleno, isósceles o equilátero.